File: fix_escapes.py
import io

with open('src/content/features/player/video-filters.js', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('\\`', '`').replace('\\${', '${')

with open('src/content/features/player/video-filters.js', 'w', encoding='utf-8') as f:
    f.write(content)

File: fix_escapes.js
const fs = require('fs');

const path = 'src/content/features/pages/subscriptions/folder-ui.js';
let txt = fs.readFileSync(path, 'utf8');

// The file was corrupted by double-escaping backticks and dollar signs.
// Let's replace literally `\`` with ``` and `\${` with `${`
const newTxt = txt
    .replace(/\\`/g, '`')
    .replace(/\\\${/g, '${');

fs.writeFileSync(path, newTxt);
console.log('Fixed literal escapes.');
